website/versioned_sidebars/version-0.11-sidebars.json

```diff
@@ -99,6 +99,10 @@
         {
```

Comment:
Drop this change, we should not change 0.11 unless we actually backport the plugin to 0.11.

Reply:
And you should update the sidebar in website (the 'next' version).